src/stores/products/index.ts

```diff
@@ -0,0 +1,20 @@
+import { createReduxStore } from '@wordpress/data';
+import { reducer } from './reducer';
+import { selectors } from './selectors';
+import { resolvers } from './resolvers';
+import { actions } from './actions';
+import { State } from '@dokan/definitions/dokan-product';
+
+export const storeName = 'dokan/products';
+
```

Comment:
-  Extract the store as like [WooCommerce](https://github.com/woocommerce/woocommerce/blob/b2cb8e5952396e5d1bc2b704c441bf88324651ec/packages/js/data/src/orders/constants.ts#L1)
- Store Name should be accessible like https://github.com/getdokan/dokan/blob/6d9c7789170d7e6e90cdc4d34132574817fa1895/src/vendor-dashboard/reports/analytics/components/leaderboard/index.js#L15